contrib/debian/manpages/zcash-cli.1

```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
-.TH ZCASH-CLI "1" "January 2017" "Zcash RPC client version v1.0.5" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.4.
+.TH ZCASH "1" "February 2017" "Zcash RPC client version v1.0.6" "User Commands"
```

Comment:
Title needs to be reverted to `ZCASH-CLI` (this is a manual step for now, unfortunately).